packages/k/keystone/xmake.lua

```diff
@@ -0,0 +1,34 @@
+package("keystone")
+    set_homepage("http://www.keystone-engine.org")
+    set_description("Keystone assembler framework: Core (Arm, Arm64, Hexagon, Mips, PowerPC, Sparc, SystemZ & X86) + bindings")
+    set_license("GPL-2.0")
+
+    add_urls("https://github.com/keystone-engine/keystone/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/keystone-engine/keystone.git")
+
+    add_versions("0.9.2", "c9b3a343ed3e05ee168d29daf89820aff9effb2c74c6803c2d9e21d55b5b7c24")
+
+    add_deps("cmake")
+
+    if is_plat("windows") then
+        add_syslinks("shell32")
+    end
+
+    on_load(function (package)
+        package:add("PATH", "bin")
```

Comment:
```lua
if package:is_plat("windows") and package:config("shared") then
    package:add("defines", "LM_EXPORT")
end
```

Reply:
I think it should refer to static not to shared.